metadata-service/factories/src/main/java/com/linkedin/gms/factory/search/ElasticSearchServiceFactory.java

```diff
@@ -49,6 +47,7 @@ protected ElasticSearchConfiguration elasticSearchConfiguration(
   }
 
   @Bean
+  @Nullable
```

Comment:
Ah neat, thought you had to do required = false, but looks like Spring will recognize `@Nullable` too

Reply:
This means something slightly different. It means that we can inject a null but something is required. Slightly different semantics, required but nullable. The reason is the injection of the variable as a parameter it seems.